### ISSUE TITLE
Console Log Spam Delays Rendering of Table Data

### ISSUE THREAD

**Issue by busches** [gregnb/mui-datatables#1873]
Upgrading from 3.8.2 to 3.8.3 results in a lot of Console.log spam:

![image](https://user-images.githubusercontent.com/857700/155010643-ed4a81b3-9c14-4adc-98ce-d15bbbeb9d2c.png)

While annoying, this normally wouldn't be a problem, but since the spam is taking so long to output, it's delaying the actual data in the table from loading, which is a problem for users and Cypress tests.

Appears to be directly related to the changes here: https://github.com/gregnb/mui-datatables/pull/1825

## Expected Behavior
No `console.log` spam

## Current Behavior
Lots of `console.log` spam.

## Steps to Reproduce (for bugs)
Happens on any usage of a `MUIDataTable` from what I can tell.

Example stack trace from the logs:

```
    console.log
      {
        'data-testid': 'MuiDataTableBodyCell-1-0',
        dataIndex: 0,
        rowIndex: 0,
        colIndex: 1,
        columnHeader: 'Tax Rules Assigned',
        print: true,
        options: {
          caseSensitive: false,
          consoleWarnings: true,
          disableToolbarSelect: false,
          download: true,
          downloadOptions: { filename: 'tableDownload.csv', separator: ',' },
          draggableColumns: { enabled: false, transitionTime: 300 },
          elevation: 4,
          enableNestedDataAccess: '',
          expandableRows: false,
          expandableRowsHeader: true,
          expandableRowsOnClick: false,
          filter: false,
          filterArrayFullMatch: true,
          filterType: 'dropdown',
          fixedHeader: true,
          fixedSelectColumn: true,
          pagination: true,
          print: false,
          resizableColumns: false,
          responsive: 'vertical',
          rowHover: true,
          rowsPerPageOptions: [ 10, 15, 100 ],
          search: true,
          selectableRows: 'none',
          selectableRowsHideCheckboxes: false,
          selectableRowsOnClick: false,
          selectableRowsHeader: true,
          serverSide: false,
          serverSideFilterList: null,
          setTableProps: [Function: setTableProps],
          sort: true,
          sortFilterList: true,
          tableBodyHeight: 'auto',
          tableBodyMaxHeight: null,
          sortOrder: {},
          textLabels: {
            body: [Object],
            pagination: [Object],
            toolbar: [Object],
            filter: [Object],
            viewColumns: [Object],
            selectedRows: [Object]
          },
          viewColumns: false,
          selectToolbarPlacement: 'replace',
          tableId: '049594251753001384',
          title: ''
        },
        tableId: '049594251753001384',
        isEmpty: false,
        children: 2
      }

      at TableBodyCell (node_modules/mui-datatables/src/components/TableBodyCell.js:115:41)

    console.log
      isEmtyfalse

      at TableBodyCell (node_modules/mui-datatables/src/components/TableBodyCell.js:176:18)
```

## Your Environment

| Tech         | Version |
|--------------|---------|
| Material-UI  |   4.12.3      |
| MUI-datatables  |     3.8.3    |
| React        |      17.0.2   |
| browser      |     Chrome, Firefox, and Electron all had the same issue    |
| etc          |         |

**Comment by busches** [comment]
Waiting on new version

**Comment by mhorger3** [comment]
@wdh2100 do you know when the next target V3 release is tentatively scheduled for?

**Comment by wdh2100** [comment]
@rodrigomercadorynb  @busches @mhorger3 

sorry. I'll release the new version(v3) after work.

**Comment by wdh2100** [comment]
fixed 3.8.5

**Comment by rodrigomercadorynb** [comment]
> fixed 3.8.5

Can confirm it´s fixed!! 
Thank youu 😀😀
